scripts/gulpfiles/build_tasks.js

```diff
@@ -753,17 +741,23 @@ function format() {
       .pipe(gulp.dest('.'));
 }
 
-module.exports = {
-  prepare: prepare,
-  build: build,
-  javaScriptAndDeps: buildJavaScriptAndDeps,
-  javaScript: buildJavaScript,
-  deps: buildDeps,
-  generateLangfiles: generateLangfiles,
-  langfiles: buildLangfiles,
-  compiled: buildCompiled,
-  format: format,
-  checkinBuilt: checkinBuilt,
-  cleanBuildDir: cleanBuildDir,
-  advancedCompilationTest: buildAdvancedCompilationTest,
-}
+// Main sequence targets.  Each should invoke any immediate prerequisite(s).
+exports.cleanBuildDir = cleanBuildDir;
+exports.messages = buildLangfiles;
+exports.tsc = buildJavaScript;
```

Comment:
Is there any reason not to have the exported identifiers match the internal identifiers? Seems like an annoying level of indirection.

Reply:
I agree that as-is this is annoying.  My plan was to update the names of the npm scripts to match, but hadn't actually done that yet.

Do you have an opinion about changing the names of the scripts to match the tasks, i.e.:

`npm run tsc` instead of `… build:javascript`, `… deps` instead of `build:deps`, etc.

  If you don't like these names then I will revert the task renames for now.

Reply:
Changing the names of the scripts to match the tasks sgtm! I would just like them to be matchy-matchy and descriptive, I don't care what direction changes in hehe.